src/flowTypes.js

```diff
@@ -420,7 +420,10 @@ export type Task = {
         sha1: string,
         name: string
     },
-    due_at: ?string
+    createdAt: string,
```

Comment:
should use the types as the API returns
created_at, due_at etc.

Reply:
Separated API response types and the types used by components